lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -258,6 +258,9 @@ module InternalConstants
     config_param :zone, :string, :default => nil
     config_param :vm_id, :string, :default => nil
     config_param :vm_name, :string, :default => nil
+    # Kubernetes-specific parameters.
+    config_param :k8s_cluster_name, :string, :default => nil
+    config_param :k8s_location, :string, :default => nil
```

Comment:
We should add a comment saying that this is the master's location, or whatever it actually is, to ensure they're not setting this to be the node's location.